Mathlib/Algebra/Order/Group/Basic.lean

```diff
@@ -28,9 +28,6 @@ lemma zpow_right_strictMono (ha : 1 < a) : StrictMono fun n : ℤ ↦ a ^ n := b
 
 @[deprecated (since := "2024-09-19")] alias zsmul_strictMono_left := zsmul_left_strictMono
 
-@[to_additive zsmul_pos] lemma one_lt_zpow (ha : 1 < a) (hn : 0 < n) : 1 < a ^ n := by
-  simpa using zpow_right_strictMono ha hn
```

Comment:
Where is this one gone?

Reply:
To this file
Mathlib/Algebra/Order/Monoid/Unbundled/Pow.lean
where it is stated in a slightly greater generality.

Reply:
Ah, right! Sorry for the noise